cves/CVE-2017-9805.yml

```diff
@@ -45,7 +45,11 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  CVE-2017-9805 is a security vulnerability that could allow the execution of
+  arbitrary code through XML deserialization. When the vulnerable module
```

Comment:
Not sure if "XML deserialization" is too technical, but if so you could change it to a more general description about input which had to be interpreted/parsed through to make classes.

Reply:
simplify into what XML deserialization means